docs/conf.py

```diff
@@ -128,10 +128,15 @@ def setup(app):
     ("py:class", "ctapipe.compat.StrEnum"),
 ]
 
-# temporary fixes to ignore reference warnings and ensure build
+# temporary workaround to ignore reference warnings and ensure build
 nitpick_ignore += [
     ("py:obj", "ctapipe.calib.CameraCalibrator"),
     ("py:obj", "ctapipe.calib.GainSelector"),
+    ("py:obj", "CameraGeometry"),
```

Comment:
This is likely just somewhere in the docs missing the FQN. So instead of ignoring a broken ref to `CameraGeometry` here, the reference should be fixed to something like `~ctapipe.instrument.CameraGeometry`